README.md

```diff
@@ -60,15 +61,27 @@ Un piccolo gruppo della community GameLoop ha seguito il corso nel 2018, potete
 
 > Sentendo altri mi è sembrato un settore molto influenzato dalle raccomandazioni. Da persona esterna con nessuna esperienza di lavoro nell'ambito, come ci si entra?
 
-...
+Trattandosi di un settore molto specialistico, dove cioè si richiedono spesso figure con molta esperienza in un campo ristretto, è facile che si ricorra al networking, ovvero ad un giro di conoscenze e raccomandazioni. Questo non significa per forza che sia difficile entrare nel settore di per sé, ma solamente che non è immediato diventare una figura ritenuta esperta in un determinato campo.
+Come scritto sopra, un portfolio può aiutare a dimostrare a quale campo specialistico si è interessati.
 
 > Come raggiungo gli standard richiesti dall'industria? Non solo da autodidatta, ma anche mantenendo l'ago della bilancia orientato verso la creatività (che è diversa e originale da persona a persona), senza ricadere nella normalizzazione spersonalizzante che spesso viene richiesta, in nome di un alto livello produttivo, dal settore dell'intrattenimento digitale in generale (a mio parere).
 
-...
+La spersonalizzazione derivante dall'alto livello di specializzazione può essere evitata puntando a studi di sviluppo meno massivi e più indipendenti, ovvero evitando quelli che generalmente si considerano studi AAA favorendo invece chi è al lavoro sui cosiddetti giochi "indie".
```

Comment:
Buona risposta! La spersonalizzazione è una questione soggettiva no? Immagino dipenda da vari fattori, quindi magari aggiungerei qualcosa per far capire che si può lavorare in una grossa azienda senza soffrire questo aspetto.

Reply:
Sì, la risposta potrebbe essere sviluppata oltre. Ci penso meglio.

Reply:
Ho riscritto la risposta in un secondo commit.

Reply:
Perfetta, ora non lascia spazio a dubbi, grazie per il contributo!